x/account/keeper/msg_server_create_coordinator.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/x/account/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k msgServer) CreateCoordinator(
+	goCtx context.Context,
+	msg *types.MsgCreateCoordinator,
+) (*types.MsgCreateCoordinatorResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Validate the coordinator address
+	_, err := sdk.AccAddressFromBech32(msg.Address)
+	if err != nil {
+		return &types.MsgCreateCoordinatorResponse{},
+			status.Error(codes.InvalidArgument,
```

Comment:
For errors, we should use the standard package of Cosmos SDK for this: https://github.com/cosmos/cosmos-sdk/tree/master/types/errors

When we have custom error that can be recurrent, we should register them ourself: https://github.com/tendermint/spn/blob/master-legacy/x/launch/types/errors.go